elastic/kibana
# [Infrastructure UI] Show processes on a single hosts

Closes #150907

## Summary
This PR adds the Processes tab to the single host flyout. The component is already implemented in Inventory so we can reuse it here 

## Testing
- Go to hosts view
- Open the flyout for any host to see the single host details
- Click on the processes tab 
⚠️ If you want to see the processes summary on top (where the total processes are displayed) you need inside your metricbeat modules yml configuration to include the `process_summary` so your config should include:
```
metricbeat.modules:
  - module: system
    metricsets:
      ..... other data  .......
      - process # Per process metrics
      - process_summary # Process summary
    ..... other data  .......
```
<img width="1913" alt="image" src="https://user-images.githubusercontent.com/14139027/228534978-c38437e4-4279-4ad4-9fc8-5222cbd15c2e.png">
